SU2_CFD/src/variables/CIncNSVariable.cpp

```diff
@@ -69,6 +68,11 @@ bool CIncNSVariable::SetPrimVar(unsigned long iPoint, su2double eddy_visc, su2do
 
   FluidModel->SetTDState_T(Temperature, scalar);
 
+  /*--- flamelet block ---*/
+  Solution(iPoint,nDim+1) = FluidModel->GetTemperature();
+  Temperature = Solution(iPoint,nDim+1);
+  const auto check_temp = SetTemperature(iPoint, Temperature);
```

Comment:
How does this work? Use the old temperature then compute a new temperature :thinking:

Reply:
The only thing we need to do here is to get temperature from the lookup table (SetTdState_T) and then put it into Solution(iPoint,indices.Temperature()). Changed.